contracts/RMRK/utils/RMRKBulkWriter.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.18;
+
+import "../equippable/IERC6220.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+error RMRKCanOnlyDoBulkOperationsOnOwnedTokens();
+error RMRKCanOnlyDoBulkOperationsWithOneTokenAtATime();
```

Comment:
I'd extract this to the errors library. I guess there is an argument to be made for keeping the `util` library errors contained there, but on the other hand, it is convenient to have all of the errors in one place 🤔 

TL, DR;  I'd move the errors to the library 😅

Reply:
Yeah I am more biased towards keeping errors for utils in the utils. But most of them are already on the lib so I will listen to you :grimacing: